### PR TITLE
Bug 1844944: Add default labels to default sc

### PR DESCRIPTION
**Pull request by yaacov**
Add a "(default)" string to the default storage class option.

Screesnshots:
Before:
Screenshot from 2020-08-09 10-56-06

After:
Screenshot from 2020-08-09 11-09-19

### CODE REVIEW

**Comment by yaacov** [comment]
@matthewcarleton please review

File: frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx [hunk 1]
```diff
@@ -539,16 +541,30 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
             </FormSelect>
           </FormRow>
           {source.requiresStorageClass() && (
-            <K8sResourceSelectRow
+            <FormSelect
```
Comment (glekner) [inline frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx:544]: This change works but you lose the value that comes with `K8sResourceSelectRow` which is the loading state.
I suggest adding a generic `defaultAnnotation` prop to `K8sResourceSelectRow` and update the component to look for it in the annotations and if its there add `(default)` to the name

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx]: :+1: done

**Comment by glekner** [comment]
nice!
/lgtm

**Pull request merged**